CRUD.c

```diff
@@ -0,0 +1,186 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+
+typedef struct {
+    int id;
+    char name[50];
+    int age;
+} User;
+
+#define FILE_NAME "users.txt"
+
+
+void initializeFile() {
+    FILE *file = fopen(FILE_NAME, "a");
+    if (file == NULL) {
+        printf("Error creating the file!\n");
+        exit(1);
+    }
+    fclose(file);
+}
+
+
+void createUser() {
```

Comment:
The requirement was to have a unique user ID. The code doesn't seem to consider and fulfill this requirement.